sqrl-execute/sqrl-execute-http/sqrl-execute-http-vertx/src/main/java/com/datasqrl/graphql/type/SqrlVertxScalars.java

```diff
@@ -0,0 +1,41 @@
+package com.datasqrl.graphql.type;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import graphql.Scalars;
+import graphql.schema.Coercing;
+import graphql.schema.GraphQLScalarType;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import lombok.SneakyThrows;
+
+public class SqrlVertxScalars {
+
+  static final ObjectMapper objectMapper = new ObjectMapper();
+
```

Comment:
This is where we would add a scalar type for vectors as well, right? Do we have a ticket for that somewhere?

Reply:
Yeah, but I have it so it skips the field during graphql generation if it can't find the associated type (with a log line). A use can manually create the graphql field with a double array type and it would work.